introduction.md

```diff
@@ -0,0 +1 @@
+aaa bbbb ccccc dddd eee fff ggg  hhhh iii jjj kkk lll mmm nnn ooo pp qqq rrr ss ttttt uuu vvv wwww xx yyyy zzz
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.